catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ExportCommand.java

```diff
@@ -115,8 +117,7 @@
     String output = Paths.get(System.getProperty("ddf.home"), FILE_NAMER.get())
             .toString();
 
-    @Option(name = "--delete", required = true, aliases = {
-            "-d"}, multiValued = false, description = "Delete Metacards and content after export. EG: --delete=true or --delete=false")
+    @Option(name = "--delete", required = true, multiValued = false, description = "Delete Metacards and content after export. EG: --delete=true or --delete=false")
```

Comment:
Purposely made it required so the user has to put it every time. You still can use `--delete` as the short implicit form of `--delete=true`.